test/format-test.cc

```diff
@@ -948,7 +948,14 @@ TEST(format_test, precision) {
   EXPECT_THAT(outputs,
               testing::Contains(fmt::format("{:.838A}", -2.14001164E+38)));
 
+#if FMT_CLANG_VERSION
+#  pragma clang diagnostic push
+#  pragma clang diagnostic ignored "-Wliteral-range"
   auto ld = 8.43821965335442234493E-4933L;
+#  pragma clang diagnostic pop
+#else
+  auto ld = 8.43821965335442234493E-4933L;
+#endif
   EXPECT_EQ(fmt::format("{:.0}", ld), ld != 0 ? "8e-4933" : "0");
```

Comment:
Let's use `std::numeric_limits<long double>::min()` and wrap `EXPECT_EQ` in `std::numeric_limits<long double>::digits == 64` check instead. This way we won't need to suppress anything.

Reply:
On x86 CI:
```
Expected equality of these values:
  fmt::format("{:.0}", ld)
    Which is: "3e-4932"
  ld != 0 ? "8e-4933" : "0"
    Which is: "8e-4933"
```

I replaced "8e-4933" with "3e-4932". Is this the correct replacement?

Reply:
Yeah, 3e-4932 is correct. For this test the exact value doesn't matter, just needs to be small.